app/src/main/java/org/mozilla/fenix/utils/Settings.kt

```diff
@@ -887,6 +887,14 @@ class Settings(private val appContext: Context) : PreferencesHolder {
         default = true
     )
 
+    /**
+     * Indicates if the onboarding features should be shown.
+     */
+    var shouldShowOnboardingFeatures by booleanPreference(
+        appContext.getPreferenceKey(R.string.pref_key_should_show_onboarding_features),
+        default = FxNimbus.features.homescreen.value().onboarding
```

Comment:
Can you implement this similar to `showRecentTabsFeature`? I don't think we want to call `FxNimbus.features.homescreen.value()` directly since it can throw an exception.

Reply:
I am also expecting that we should have something in `FeatureFlags` so that we can manually toggle the pref on for our local testing.

Reply:
I've been working on Nimbus recently - it's safe to call `FxNimbus.features.homescreen.value()` directly. It doesn't throw an exception any more.

You don't need to pass it a context anymore too (which was part of the cause of the problem then.